duckduckgo/iOS
# Migrate legacy saved searches and favourites

Reviewer: Mia
Asana: https://app.asana.com/0/392891325557410/390989879439961
CC: Caine

**Description**:
Migrates the users saved searches and stories from the Search and Stories app in to bookmarks in the new app. 

Tested with tagged version of 5.0.1 (delete the mogenerator target and dependency, comment out references to test flight, and update the bundle id to match the new app)

Tested with tagged version of 6.0.2 (update the bundle id to match the new app)

Tested with tagged version of 6.0.9 (update the bundle id to match the new app)


**Steps to test this PR**:
1. Checkout a version of the Search and Stories app to test against, as in the description above
2. Launch the Search and Stories app from Xcode 
3. Favourite one or more stories and make a note of what they are
4. Favourite one or more searches and make a note of what they are
5. Launch the new app from Xcode
6. Finish the on-boarding screens and check the favourited stories and searches are present in the bookmarks


###### Reviewer Checklist:


###### PR DRI Checklist:
